### ISSUE TITLE
Milne open textbooks

### ISSUE THREAD

**Issue by Popolechien** [openzim/zim-requests#1135]
- Website URL: https://milneopentextbooks.org/
- License: **Creative Commons** 
- Desired ZIM Title: **Milne Open Textbooks**
- Desired ZIM Description: **A catalog of textbooks authored and peer-reviewed by SUNY faculty and staff** (80 characters or less)**
- Desired ZIM Icon –png (URL or attach one): https://milneopentextbooks.org/wp-content/uploads/Milne-Open-textbook-logo_red-blue-tan-cropped.png
- Language (ISO 639-3): **eng**
- Is this a MediaWiki?: **no**

**Comment by Rexadev** [comment]
@Popolechien are they related https://github.com/openzim/zim-requests/issues/1051 ?

**Comment by Popolechien** [comment]
Yep, looks like they're very related. I'd favour this one as the publishing website seems to be more about the publishing house and trying to access the books listed sends us back to milnesopentextbook

**Comment by RavanJAltaie** [comment]
File is ready at the library
https://library.kiwix.org/viewer#milneopentextbooks.org_en_all_2024-08

**Comment by Rexadev** [comment]
@Popolechien they are different see https://milnepublishing.geneseo.edu/catalog/ https://milneopentextbooks.org/booktype/ost-books/
